### DISCUSSION TITLE
v0.2.0

### DISCUSSION THREAD

**Discussion by contentis** [NVIDIA/Stable-Diffusion-WebUI-TensorRT#218] [Announcements]
## Features

- Faster LoRA checkpoint conversion.
- Combining multiple LoRAs at runtime.
- Use prompt embedding for LoRA.
- SDXL LoRA support.
- Automatic engine selection.
- Improved HIGHRES.FIX engine selection
- PyTorch fallback.
- Print the correct profile is loaded. 

## Minor

- Use script callback to handle inference.
- External data format for ONNX
- Deleted dead code and duplications

<hr /><em>This discussion was created from the release <a href='https://github.com/NVIDIA/Stable-Diffusion-WebUI-TensorRT/releases/tag/v0.2.0'>v0.2.0</a>.</em>

**Comment by wywywywy** [comment]
Congrats on getting a shoutout during CES. Well done team.

Once this supports ControlNet (0.3.0?) I know many of us will start using it full time!

**Reply by T9es** [reply]
Definitely, a lot of people waiting for that. There's also a lot of people waiting for mass engine export with loRAs. Sadly, the 0.2.0 release is really buggy and unstable, even unusable for people.
